botcamp_sql_Creates.sql

```diff
@@ -0,0 +1,58 @@
+
+CREATE DATABASE highschool;
+USE highschool;
+CREATE USER sqluser IDENTIFIED BY 'sqluserpw';
+
+grant usage on *.* to sqluser@localhost identified by 'sqluserpw';
+grant all privileges on feedback.* to sqluser@localhost;
+
+CREATE TABLE Student(
+	registration_number	INT	NOT NULL AUTO_INCREMENT,

```

Comment:
Be consistent about the naming convention. If you use underscore as separator or you use CamelCase, please use the same for all the fields.